README.md

```diff
@@ -5,39 +5,30 @@ Widget for visualizing 3D Radia models in a jupyter notebook
 
 RadiaViewer allows users to render a magnet geometry and solve for its fields in a self-contained widget. The rendering is done by the VTK.js library.
 
-**NOTE**: The current build requires jupyterlab 2.1.0 and jupyterlab-server 1.2.0. 
+**NOTE**: This depends on JupyterLab 3.x
 
 Installation
 ------------
 
 To install use pip:
 
     $ pip install .
-    $ jupyter nbextension install --py --symlink --sys-prefix jupyter_rs_radia
-    $ jupyter nbextension enable --py --sys-prefix jupyter_rs_radia
-
-To install for jupyterlab
-
-    $ jupyter labextension install js
+	$ jupyter labextension install .
 
 For a development installation (requires npm),
 
     $ git clone https://github.com/radiasoft/jupyter-rs-radia.git
     $ cd jupyter-rs-radia
     $ pip install -e .
-    $ jupyter nbextension install --py --symlink --sys-prefix jupyter_rs_radia
-    $ jupyter nbextension enable --py --sys-prefix jupyter_rs_radia
-    $ jupyter labextension install js
+    $ jlpm install:extension
 
-When actively developing your extension, build Jupyter Lab with the command:
+When actively developing your extension, use watch to hot reload on changes:
 
-    $ jupyter lab --watch
+    $ jlpm watch
```

Comment:
jlpm watch also failed:

```
[py3;@v4 jupyter-rs-radia]$ jlpm watch
yarn run v1.21.1
$ watch:labextension
/bin/sh: watch:labextension: command not found
error Command failed with exit code 127.
```

I don't know if that is related to the previously failed command